src/pages/Index.vue

```diff
@@ -141,8 +158,21 @@ export default {
     ProfileTab,
     RepositoryTab,
     ApplicationTab,
+    KUploader,
   },
 
-  methods: {},
+  methods: {
+    getUploadedFileUrl(url) {
+     this.form.avatar = url
+     this.form['avatar'] === this.form.avatar
+     this.dialogPhoto = false
```

Comment:
The avatar is assign to local variables, but its not submitted to backend to be store.